src/opnsense/service/conf/actions.d/actions_unbound.conf

```diff
@@ -83,3 +83,10 @@ command:pgrep -q -nF /var/run/unbound.pid && echo "unbound is running" || echo "
 parameters:
 type:script_output
 message:request Unbound status
+
+[reconfigure]
+command:pluginctl -s unbound restart
+parameters:
+type:script
+message:Configuring and restarting Unbound
+description:Configure and restart Unbound via legacy code
```

Comment:
any reason we need a cron job (description set)?

Reply:
We don't, misinterpretation on my end :)